test/helpers.js

```diff
@@ -6,16 +6,24 @@ import * as babel from 'babel-core';
 
 const fixtures = path.join(__dirname, 'fixtures');
 
-export const transform = (filename: string): Promise<string> => {
+export const babelNoModules = {
+  presets: [ ['env', { modules: false, targets: { node: 'current' } }] ],
+};
+
+export const transform = (filename: string,
+                          babelOptionOverrides: any): Promise<string> => {
```

Comment:
I guess I don't have the linter configured for quite so many styles, but if you wouldn't mind changing the style to:

```js
export const transform = (
  filename: string,
  babelOptionOverrides: { [string]: mixed },
): Promise<string> => {
  /* ... */
}
```